src/PooledArrays.jl

```diff
@@ -191,6 +192,12 @@ Base.copyto!(dest::PooledArray{T, R, N, RA},
              src::PooledArray{T, R, N, RA}) where {T, R, N, RA} =
     copyto!(dest, 1, src, 1, length(src))
 
+# TODO: handle case when dest is SubArray
+
+Base.copyto!(dest::PooledArray{T, R, N, RA},
+             src::SubArray{T, N, PooledArray{T, R, M, RA}}) where {T, R, N, M, RA} =
+    copyto!(dest, 1, src, 1, length(src))
```

Comment:
Doesn't the `AbstractArray` fallback already do this? Same for the method above BTW.

Reply:
OK